src/MongoDB.Driver.Core/Core/Connections/TcpStreamFactory.cs

```diff
@@ -48,7 +48,11 @@ public TcpStreamFactory(TcpStreamSettings settings)
         // methods
         public Stream CreateStream(EndPoint endPoint, CancellationToken cancellationToken)
         {
-#if NETSTANDARD1_5 || NETSTANDARD2_0
+#if NET452
```

Comment:
this is the same problem as It was solved here: https://github.com/vincentkam/mongo-csharp-driver/pull/117

Reply:
My first reaction was that it's better to have my approach since we need to only exlude `NET452` and I reversed these code blocks just to avoid using `!`(negation). But ability to easier search these TFs in the code, if we want to decide whether the same change should be added for the next TF or not, makes sense to me. I think in some next frameworks, the method `ResolveEndPointsAsync` can not be required, even though now it has to be added

Reply:
done